x/schema/queue.go

```diff
@@ -27,6 +27,26 @@ func GenerateCreateTableQuery(queueName string) string {
 	`, quotedTableName, quotedTableName[1:len(quotedTableName)-1])
 }
 
+// GenerateCreateTableQueryCompositeIndex returns the query for creating the queue table
+func GenerateCreateTableQueryCompositeIndex(queueName string) string {
```

Comment:
Could we keep it by the test? This package used to be for testing, but then @kedlas wanted  it to be exported to the world.

Reply:
done